int16 theta1
int16 theta2
int16 theta3
int16 theta4
int16 theta5
int16 theta6
